packages/jsActions/nanoflow-actions-native/typings/mx.d.ts

```diff
@@ -6,4 +6,8 @@ declare namespace mx {
     interface data extends mx.data {
         update: (param: { guid?: string | undefined; entity?: string | undefined; callback?: () => void }) => void;
     }
+    interface session extends mx.session {
+        clearCachedSessionData: () => Promise<void>;
+    }
+    function reload(): void;
```

Comment:
This is causing issues (the build issues). I can't say why yet.

Reply:
There seems to be a clash with the base definition `/node_modules/mendix-client/index.d.ts`

Reply:
Fixed! ✅